xpra/server/window/window_video_source.py

```diff
@@ -457,13 +461,13 @@ def lossless(reason):
         if self.is_tray:
             return nonvideo(100, "system tray")
         text_hint = self.content_type=="text"
-        if text_hint and not TEXT_USE_VIDEO:
+        if text_hint and not TEXT_USE_VIDEO and not (WINDOW_FOCUS_VIDEO_ENCODER and self.has_focus):
```

Comment:
What is the benefit of using nvenc for text?
Where is `TEXT_USE_VIDEO` defined?

Reply:
I believe `TEXT_USE_VIDEO = envbool("XPRA_TEXT_USE_VIDEO", False)` was already there, so it's nothing new - however the goal here (as also mentioned above) was to push nvenc to be used as much as possible for a window in focus, and things like text content-type would override that and force non-video